app/helpers/shop_helper.rb

```diff
@@ -52,4 +52,10 @@ def show_shopping_cta?
 
     true
   end
+
+  private
+
+  def show_groups_tabs
+    current_distributor.groups.any? && !current_distributor.hide_groups_tab?
```

Comment:
Checking the setting first may mean that we don't have to query the database for groups. But it may also be a useless micro-optimisation on my part. :shrug: 
```suggestion
     !current_distributor.hide_groups_tab? && current_distributor.groups.any?
```

Reply:
I think it's a good idea. The `hide_groups_tab` value has already been loaded into memory, so it makes sense to check that first.